src/test/java/de/tum/in/www1/artemis/OrganizationIntegrationTest.java

```diff
@@ -130,7 +130,7 @@ void testRegisterForCourseWithOrganizationsEnabled() throws Exception {
         updatedStudent = request.postWithResponseBody("/api/courses/" + course2.getId() + "/register", null, User.class, HttpStatus.OK);
         assertThat(updatedStudent.getGroups()).as("User is registered for course").contains(course2.getStudentGroupName());
 
-        request.postWithResponseBody("/api/courses/" + course3.getId() + "/register", null, User.class, HttpStatus.BAD_REQUEST);
+        request.postWithResponseBody("/api/courses/" + course3.getId() + "/register", null, User.class, HttpStatus.FORBIDDEN);
```

Comment:
If I understand it correctly, this test is about a user trying to register in a course from another organization. I would argue that a error status of 403 is fitting because it is forbidden to access the resource of another organization. What do you think?